<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="data_widget" native="true">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Max Frequency</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="max_freq_line"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="fft_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="plot_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="spectrogram_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_save_plots"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_save_plots">
   <property name="text">
    <string>Save plots</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
